Donation-platform/src/main/webapp/frontend/src/getTagScores.js

```diff
@@ -0,0 +1,114 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+///home/anerishah/step-donation/step28-2020/Donation-platform/src/main/webapp/frontend/src
+///usr/local/nvm/versions/node/v10.14.2/bin/node
+
+var googleTrends = require('/home/anerishah/step-donation/step28-2020/Donation-platform/src/main/webapp/frontend/node_modules/google-trends-api/lib/google-trends-api.min.js');
+const fetch = require("node-fetch");
+
+//sample method to demonstrate GET request
+function sendQuery() {
+    fetch('https://donations-step-2020-new.uc.r.appspot.com/tag-query', {method: 'GET'})
+    //fetch('https://8080-5a8baf4b-def3-4b41-83c1-47f3b4af6cdc.us-west1.cloudshell.dev/tag-query', {method: 'GET'})
```

Comment:
What is the purpose of this comment? Maybe add some reasonings here.

Reply:
removed the comment. was for my purpose of keeping track of the local URL.